movie-blog/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import CreateElement from "./components/create-element";
+import ComponentElement from "./components/component";
+import PureComponentElement from "./components/pure-component";
+import FunctionalComponentElement from "./components/functional-component";
+
+const createElement = React.createElement(
+  'h1',
+  null,
+  'Hello World!'
+)
+
+function App() {
+  return (
```

Comment:
Consider passing as props the messages you need to render. This way, your components will be more flexible and re-usable.